Makefile

```diff
@@ -11,10 +11,12 @@ CFLAGS := -Werror -Wall -Wextra -Wpedantic -g -I src/
 PROFILE := release
 DESTDIR=/usr/local
 
+ifeq ($(PROFILE), debug)
 ifeq ($(CC), clang)
```

Comment:
This falls away in the next commit because we drop the `ifeq` conditional on `$(CC)` but jeez was it depressing to learn that GNU Makefiles don't support boolean conditionals and folks suggest madness like [concatenating your conditions](https://stackoverflow.com/a/6452937) :scream: